client/verta/verta/endpoint/_endpoint.py

```diff
@@ -435,7 +435,7 @@ def _create_update_body(strategy, resources=None, autoscaling=None, env_vars=Non
 
         """
         if not isinstance(strategy, _UpdateStrategy):
-            raise TypeError("`strategy` must be an object from verta.endpoint.strategies")
+            raise TypeError("`strategy` must be an object from verta.endpoint.update.strategies")
```

Comment:
I think it's `verta.deployment.update._strategies` =))

Reply:
replaced